app.py

```diff
@@ -0,0 +1,139 @@
+from flask import Flask, request, jsonify, make_response
+from flask_sqlalchemy import SQLAlchemy
+from os import environ
+from abc import ABC, abstractmethod
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = environ.get('DATABASE_URL')
+db = SQLAlchemy(app)
+
+
+class User(db.Model):
+    __tablename__ = 'users'
```

Comment:
possibly there is space to include some optimization for db queries? https://athelaseng.substack.com/p/-six-way-to-optimize-sqlalchemy

Reply:
topic to be discussed at the meeting